### DISCUSSION TITLE
retrieve files from github release and put it somewhere in the repo

### DISCUSSION THREAD

**Discussion by sylvainOL** [renovatebot/renovate#16082] [Suggest an Idea]
Hello,

I'm looking for the ability of retrieving a file from github release and update it somewhere.

flux is roughly doing it but with flux binary (although it could use this way of doing) and I've not seem a manager being able to do it.

Is it possible to do it?

**Comment by rarkins** [comment]
We don't have anything generic like this. If you were self hosting then you might be able to add your own post upgrade task to do it.

**Reply by sylvainOL** [reply]
yep, I was thinking about doing that, I just wanted to be sure it's available first :)

thanks for quick reply
